structures_and_records.md

```diff
@@ -15,7 +15,7 @@ structure, as is the dependent product (Sigma) type.
 In general, whenever we define a structure ``S``, we usually
 define *projection* functions that allow us to "destruct" each
 instance of ``S`` and retrieve the values that are stored in its
-fields. The functions ``prod.pr1`` and ``prod.pr2``, which return the
+fields. The functions ``prod.1`` and ``prod.2``, which return the
```

Comment:
```suggestion
fields. The functions ``Prod.fst`` and ``Prod.snd``, which return the
```
These are the actual projection names

Reply:
@lovettchris functional programmers (and mathematicians) tend to like one-liners (i.e., ,<80 chars) and short abbreviations make righting them easier. Hence all the single character abbreviations where long words would probably be more intelligible. 

In the particular case of `fst` and `snd`, there is also the nice fact that they are the same length, which makes for a more aesthetically pleasing result when aligning code across lines .They are also the abbreviations Haskell uses for the same concept: see [`fst`](https://hoogle.haskell.org/?hoogle=fst) and [`snd`](https://hoogle.haskell.org/?hoogle=snd) on Hoogle.

Reply:
ok thanks.

Reply:
fixed